src/main/java/picard/analysis/TheoreticalSensitivity.java

```diff
@@ -204,4 +204,174 @@ public int draw() {
         }
         return normalizedHistogram;
     }
-}
+
+    /**
+     * Determines if a variant would be called under the particular conditions of a given total depth, alt depth,
+     * average base qualities, allele fraction of variant and log odds threshold necessary to exceed to call variant.
+     * @param totalDepth Depth at the site to be called, both alt and ref.
+     * @param altDepth Number of alt bases at this site.
+     * @param averageQuality Average Phred-scaled quality of bases
+     * @param alleleFraction Allele fraction we are attempting to detect
+     * @param logOddsThreshold Log odds threshold necessary to exceed for variant to be called
+     * @return
+     */
+    public static boolean isCalled(int totalDepth, int altDepth, double averageQuality, double alleleFraction, double logOddsThreshold) {
+        double threshold;
+        double sumOfQualities = altDepth * averageQuality;
```

Comment:
is there a reason you want to pass in averageQuality and not sumOfQualities? that seems to be the original value and also the one you need...so why divide and then multiply?

Reply:
fixed